share/python/procedures/findif_response_utils/tensor_helper.py

```diff
@@ -0,0 +1,118 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2016 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+
+"""
+Module of helper functions for distributed ccresponse computations.
+
+Defines functions for retrieving tensors computed at displaced geometries.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+import collections
+import shelve
+import copy
+import os
+import psi4
+import p4util
+from p4const import *
+
+
+def synthesize_displaced_tensor(db, signature, row_dim):
```

Comment:
@dgasmith Currently the only use-case, roa.py, passes these lists of data into a python wrapper that converts them to `std::vector<SharedMatrix>` s and those are passed to the underlying scatter function in the ccresponse module. 

If in the future there is any manipulation done with these types of data python-side I would add some methods to parse this type of data from the output files and  return NumPy arrays for sure!